### ISSUE TITLE
Add reminders to all events of a caldav calendar

### ISSUE THREAD

**Issue by unhammer** [SimpleMobileTools/Simple-Calendar#1000]
I'm syncing an Outlook calendar via their (readonly) .ics url (and [a separate app](https://icsx5.bitfire.at/)). The .ics doesn't seem to have the 15-minutes-before reminders that are in Outlook. Would it be possible to have a feature that puts local reminders on all events of a synced calendar?

**Comment by MrAce100** [comment]
Yes please! I am having this same issue; using ICSx5 and Teamup calendar. I would greatly appreciate this addition!
